systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    start_registry --enable-delete=true reg
```

Comment:
After the registry test is simplified, this shouldn’t be necessary.

Reply:
this had done